ibc/module.go

```diff
@@ -0,0 +1,91 @@
+package ibc
+
+import (
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/runtime/configs"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/modules/base_modules"
+	"github.com/pokt-network/pocket/utility/validator"
+)
+
+var _ modules.IBCModule = &ibcModule{}
+
+type ibcModule struct {
+	base_modules.IntegratableModule
+
+	cfg    *configs.IBCConfig
+	logger *modules.Logger
+
+	// Only a single host is allowed at a time
+	host *host
+}
+
+func Create(bus modules.Bus, options ...modules.ModuleOption) (modules.Module, error) {
+	return new(ibcModule).Create(bus, options...)
+}
+
+func (m *ibcModule) Create(bus modules.Bus, options ...modules.ModuleOption) (modules.Module, error) {
+	*m = ibcModule{
+		cfg:    bus.GetRuntimeMgr().GetConfig().IBC,
+		logger: logger.Global.CreateLoggerForModule(modules.IBCModuleName),
+	}
+	m.logger.Info().Msg("🪐 creating IBC module 🪐")
+
+	for _, option := range options {
+		option(m)
+	}
+
+	bus.RegisterModule(m)
+
+	// Only validators can be an IBC host due to the need for reliability
+	actors := m.GetBus().GetUtilityModule().GetActorModules()
```

Comment:
I assumed it would be better to use this method so that once the logic is implemented to verify on the utility end, this code shouldnt have to change. Currently it is based solely off of the config file I belive.